src/Bicep.Core.Samples/Files/InvalidResources_CRLF/main.diagnostics.bicep

```diff
@@ -1501,3 +1501,9 @@ resource vnet 'Microsoft.Network/virtualNetworks@2020-06-01' = {
   }
 }
 
+resource existngResProperty 'Mock.Rp/mockExistingResource@2020-01-01' = {
+//@[28:69) [BCP081 (Warning)] Resource type "Mock.Rp/mockExistingResource@2020-01-01" does not have types available. |'Mock.Rp/mockExistingResource@2020-01-01'|

```

Comment:
>BCP081 [](start = 12, length = 6)

This isn't the diagnostic that was added (BCP167)

Reply:
Fixed